Description: Fix cleanup rule in Makefile
 also remove fsosystem-2.0.vapi and fsosystem.h
Author: Rico Rommel <rico@bierrommel.de>
--- a/fsosystem/Makefile.am
+++ b/fsosystem/Makefile.am
@@ -69,6 +69,8 @@
 CLEANFILES = \
 	$(BUILT_SOURCES) \
 	$(libfsosystem_la_VALASOURCES:.vala=.c) \
+	fsosystem.h \
+	fsosystem-2.0.vapi \
 	$(NULL)
 
 
